src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+function activeNav({ isActive }: { isActive: boolean }) {
+  return classNames('navbar-item', {
+    'has-background-grey-lighter': isActive,
+  });
+}
+
+export const Navigation = () => {
+  return (
```

Comment:
if you don't have any logic then you can return jsx without using word `return `, just in round brackets
` () => (`

(if linter allows)

check in all components